ansible/roles/owncloud/defaults/main.yml

```diff
@@ -819,6 +819,12 @@ owncloud__role_config:
     state: '{{ "present" if (owncloud__temp_path != "") else "absent" }}'
     value: '{{ owncloud__temp_path }}'
 
+# .. envvar:: owncloud__release_channel
+#
+# Release channel
+owncloud__release_channel: '{{ "stable" if (owncloud__variant == "nextcloud" and
+                                owncloud__release is version("17.0", ">="))
+                            else "production" }}'
```

Comment:
Indention should match. See the other if conditions in that file.

Also "Release channel" as variable description does not add much value to the variable name. Can you add more context? Reference to the URL would be fine. I try to limit the redundancy we have to maintain here.